README.md

```
@@ -32,6 +32,18 @@ This project provides an elegant way to remove the sora watermark in the sora2 g
   </tr>
 </table>
 
+
+**Commercial Hosted Service & Sponsorship**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)